udata/forms/fields.py

```diff
@@ -130,8 +129,8 @@ class URLField(EmptyNone, Field):
     def pre_validate(self, form):
         if self.data:
             try:
-                db.URLField().validate(self.data)
-            except db.ValidationError:
+                uris.validate(self.data)
```

Comment:
why not use the logic from `udata/models/url_field.py` here?

Reply:
This is the same logic, `udata/models/url_field.py` also use `uris.validate()` but to me this is more explicit to show the comme piece `uris.validate`

Reply:
I liked the idea of decoupling a little bit more here: rely on `URLField` that relies on `uris` in turn. But not a big problem.

Reply:
If you prefer I can, but:
- I though it was easier to find the common dependency when being explicitly written
- `db.URLField.validate()` is an instance method relying on custom field configuration. I though about giving the same parameters here too, but this is not currently in use
- model and form validation are slightly different, and I wanted to be able highlight the differences while keeping a neutral behavior in `uris.validate`

Reply:
LGTM

Reply:
:ok_hand: